### DISCUSSION TITLE
有些地方你需要继续修改的……

### DISCUSSION THREAD

**Discussion by zhongfly** [dyphire/mpv-winbuild#1] [General]
比如cache缓存，一个仓库一共只能存储5G，这使得目前仓库里只会有唯一一个缓存，光一个版本就使用3G+，这使得目前仓库里只会有唯一一个缓存，这个缓存能否在你这样依赖库细微不同的mpv编译中混用是一个问题
并且缓存在不被使用7天后就被清理的
又比如自动删除旧release，它是把所有release按照月份以及最近的tag来保留，如果你是固定把几种构建输出到不变的tag里，之后估计就会因为超过14天被删除了

**Comment by zhongfly** [comment]
关于缓存的使用，建议阅读 https://docs.github.com/cn/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#using-the-cache-action

**Comment by zhongfly** [comment]
关于自动清理release，最好看一下原版https://github.com/BtbN/FFmpeg-Builds/blob/master/util/prunetags.sh 
```shell
TAGS=( $(git tag -l "autobuild-*" | sort -r) )
```
他用这个筛选出只有autobuild-开头的tag参与清理

**Comment by dyphire** [comment]
感谢提醒，已修改。对于 cache 缓存，虽然我上午测试构建成功的版本在测试中符合预期；不过正如你所说，混用缓存是一个问题，所以我决定之后只自动构建 libass 修改版，毕竟官方版构建选择较多

**Reply by zhongfly** [reply]
还有这个修复，否则复制不了.git文件夹，切换不到gcc10分支
https://github.com/zhongfly/mpv-build/commit/e925933bd739dee0a8d579348dd7fae52db2fb93#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823

**Reply by dyphire** [reply]
Thanks
